pkgs/development/tools/java/jprofiler/default.nix

```
@@ -0,0 +1,100 @@
+{ stdenv
+, lib
+, fetchurl
+, makeWrapper
+, makeDesktopItem
+, undmg
+, jdk11 }:
```

If you need to specify a certain JDK version, you should do that in `all-packages.nix`...
```suggestion
, jdk };
```